src/main/java/org/matsim/ptFares/utils/TransitRider.java

```diff
@@ -0,0 +1,72 @@
+package org.matsim.ptFares.utils;
+
+import org.matsim.api.core.v01.Id;
+import org.matsim.api.core.v01.population.Person;
+import org.matsim.pt.transitSchedule.api.TransitStopFacility;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransitRider{
```

Comment:
Why do you implement your own transit rider and transit vehicle? Doesn't Matsim offer this out of the box?